wikipedia-dark.user.css

```
@@ -3470,10 +3470,14 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
     border-color: var(--gray-5) !important;
   }
   /*** main page ***/
-  .mw-parser-output #mp-topbanner, .mw-parser-output #mp-banner {
+  .mw-parser-output #mp-topbanner {
     border-color: var(--gray-5);
     background-color: var(--gray-2);
   }
+  .mw-parser-output #mp-banner {
+    border-color: var(--yellow-2);
+    background-color: var(--yellow-1);
+  }
```

great were nearly there =) by yellow I meant brwon which is the standard goto color for yellow replacements.
```suggestion
  .mw-parser-output #mp-banner {
    background-color: var(--brown-1);
    border-color: var(--brown-2);
  }
```

Also this needs to be moved around to this area https://github.com/StylishThemes/Wikipedia-Dark/blame/master/wikipedia-dark.user.css#L2549